recipes/sdl_ttf/all/conanfile.py

```diff
@@ -60,7 +60,7 @@ def source(self):
 
     def _patch_sources(self):
         # Handle sdl2 static target
-        if not self.options["sdl"].shared:
+        if tools.Version(self.version) < "2.0.18" and not self.options["sdl"].shared:
             tools.replace_in_file(os.path.join(self._source_subfolder, "CMakeLists.txt"),
                                   "SDL2::SDL2", "SDL2::SDL2-static")
```

Comment:
No, it's even worse in 2.0.18: https://github.com/libsdl-org/SDL_ttf/blob/release-2.0.18/CMakeLists.txt#L68-L72

They link to static SDL2 when SDL2_ttf is static, and SDL2 shared when SDL2_ttf shared. We want all to enable all combinations.

Reply:
Thank you for information.
I will fix this problem.